### PR TITLE
Add satellite layer and show map on latlon values

### PR DESCRIPTION
**Pull request by haslinghuis**
- show gps map whenever we get lat lon values instead of waiting for gps fix
- add satellite and hybrid layers.
- satellite layer is default now
- tag and help icon in image below is now removed as anyone should be familiar using maps.

ROAD:
image

SATELLITE:
image

HYBRID:
image

### CODE REVIEW

**Comment by ctzsnooze** [comment]
This is really cool!  Can't wait for it to be working properly!
At the moment I get the satellite map, and that's fantastic!
Only a couple of things to be fixed...
- clicking the buttons doesn't update the map mode
- zooming in on the Sat map appears to zoom in on the first image we get, so it gets a but blurry as I zoom in.  The image itself isn't updated, I think, by the Maps API, when we zoom in.

File: src/tabs/gps.html
Comment (chmelevskij) [inline src/tabs/gps.html:150]: This kind of drives me nuts. Why are we using links for what's clearly a button?

File: src/js/tabs/map.js
Comment (chmelevskij) [inline src/js/tabs/map.js:108]: Can we instead rely on something more stable than this? These labels can end up being translated at some point. Could add things like `value` if we'd use buttons instead of anchor tags

File: src/tabs/gps.html
Comment (haslinghuis) [reply src/tabs/gps.html:150]: Not sure - just added to the zooming `buttons`.

File: src/js/tabs/map.js
Comment (haslinghuis) [reply src/js/tabs/map.js:108]: Code suggestions please :)

File: src/tabs/gps.html
Comment (haslinghuis) [reply src/tabs/gps.html:150]: Happy to learn a better way - but our code is full of this pattern:
```
<div class="buttons">
    <a href="#" class="someClass [optionalClass]" i18n="somethingToTranslate">
</div>
```

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/tabs/gps.html
Comment (haslinghuis) [reply src/tabs/gps.html]: Added a value to see if I could read it using `e.target.value` but it was either null or undefined?

Comment (haslinghuis) [reply src/tabs/gps.html]: Fixed

File: src/js/tabs/map.js
Comment (haslinghuis) [reply src/js/tabs/map.js]: Fixed.

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
This is absolutely fantastic!  Great job @haslinghuis 

Satellite image shows every detail and makes it really easy to see the amount of wandering around as the sat fix gets better over time.

File: src/tabs/gps.html
Comment (haslinghuis) [reply src/tabs/gps.html]: https://stackoverflow.com/questions/18314019/anchor-or-button

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
